board/safety/safety_ford.h

```diff
@@ -120,6 +132,24 @@ static bool ford_get_quality_flag_valid(CANPacket_t *to_push) {
   return valid;
 }
 
+const uint16_t FORD_PARAM_LONGITUDINAL = 1;
+
+bool ford_longitudinal = false;
+
+const LongitudinalLimits FORD_LONG_LIMITS = {
+  // acceleration cmd limits (used for brakes)
+  // Signal: AccBrkTot_A_Rq
+  .max_accel = 5770,       //  2.5    m/s^s
+  .min_accel = 4230,       // -3.5    m/s^2
+  .inactive_accel = 5128,  // -0.0008 m/s^2
+
+  // gas cmd limits
+  // Signal: AccPrpl_A_Rq
+  .max_gas = 700,          //  2.5 m/s^2
+  .min_gas = 0,
+  .inactive_gas = 0,       // -5.0 m/s^2
+};
+
```

Comment:
it looks like it simplifies things a lot if we make it just gas/brake again

Reply:
the "brake signal" is in accel though, so pos/neg are wrong way around for brake